src/modules/70-pipeline/components/PipelineStudio/SavePipelinePopover/SavePipelinePopover.tsx

```diff
@@ -160,8 +169,11 @@ function SavePipelinePopover(
   })
 
   const isTemplatesEnabled = templatesFeatureEnabled && canEdit && !pipeline?.template
-
-  const isSaveDisabled = isReadonly || !isUpdated || isIntermittentLoading
+  let isSaveDisabled = isReadonly || !isUpdated || isIntermittentLoading || isMetadataUpdated
+  if (isMetadataUpdated) {
+    // Enable save even if isUpdated is false as metadata has changed
+    isSaveDisabled = false
+  }
```

Comment:
Could you have just included `!isMetadataUpdated` as the first thing?
```suggestion
  // Enable save even if isUpdated is false if metadata has changed
  const isSaveDisabled = !isMetadataUpdated || isReadonly || !isUpdated || isIntermittentLoading
```

As-is, if it gets to `|| isMetadatUpdated` it'll set to true and then you immediately flip it with the next statement.